kura/org.eclipse.kura.rest.identity.provider/src/main/java/org/eclipse/kura/internal/rest/identity/provider/v2/dto/IdentityConfigurationDTO.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Eurotech
+ *******************************************************************************/
+package org.eclipse.kura.internal.rest.identity.provider.v2.dto;
+
+public class IdentityConfigurationDTO {
+
+    private final IdentityDTO identity;
```

Comment:
Can't this be just a String representing the identity name? So we can remove `IdentityDTO`